crates/ra_db/src/lib.rs

```diff
@@ -113,7 +113,7 @@ pub trait SourceDatabase: CheckCanceled + FileLoader + std::fmt::Debug {
 }
 
 fn parse_query(db: &impl SourceDatabase, file_id: FileId) -> Parse<ast::SourceFile> {
-    let _p = profile("parse_query");
+    let _p = profile("parse_query").detail(|| format!("{:?}", file_id));
```

Comment:
For such drive-by changes, feel free to send self-approved auto PRs. I personally have a script for that: https://github.com/matklad/config/blob/master/bin/gbors

In general, I found that for larger refactorings you always discover recursive micro-refactorings, and it helps to just send those first and rebase on top optimistically.